Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Block_URL.yml

```diff
@@ -0,0 +1,724 @@
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "8"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 240,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 2c56fab1-5200-4cee-8ced-ef3ecda1a04d
+    type: condition
+    task:
+      id: 2c56fab1-5200-4cee-8ced-ef3ecda1a04d
+      version: -1
+      name: Is the URL already exist?
+      description: Is the URL already exist?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "5"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: in
+          left:
+            value:
+              simple: inputs.URL
+            iscontext: true
+          right:
+            value:
+              simple: PrismaSase.CustomURLCategory.list
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: c2cd4345-16f4-4c35-8a5d-ca15454ca1ee
+    type: regular
+    task:
+      id: c2cd4345-16f4-4c35-8a5d-ca15454ca1ee
+      version: -1
+      name: Create a new Custom URL Category
+      description: Create a new URL category.
+      script: '|||prisma-sase-custom-url-category-create'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      description:
+        simple: List of Blocked URL - XSOAR
+      folder:
+        complex:
+          root: inputs.Folder
+      name:
+        simple: List of Blocked URL - XSOAR
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+      type:
+        simple: URL List
+      value:
+        complex:
+          root: inputs.URL
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 240,
+          "y": 500
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: a1fe91ef-08cc-45c6-8881-6666ff1843d7
+    type: regular
+    task:
+      id: a1fe91ef-08cc-45c6-8881-6666ff1843d7
+      version: -1
+      name: Update the URL category
+      description: Update an existing url category.
+      script: '|||prisma-sase-custom-url-category-update'
+      type: regular
```

Comment:
Value and type are empty 
![image](https://github.com/demisto/content/assets/61933087/8d132218-a1ec-4e64-bb1d-5d180e3d7d57)

Reply:
Added the missing value. 
No need for the type - as the group should be already defined as an URL